static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -121,20 +120,12 @@ const getRequestDataFragment = ({
 
 export default function IssueAlertOptions({
   onChange,
-  alertSetting: defaultAlertSetting = RuleAction.DEFAULT_ALERT,
-  interval: defaultInterval = '1m',
-  metric: defaultMetric = MetricValues.ERRORS,
+  alertSetting = RuleAction.DEFAULT_ALERT,
+  interval = '1m',
+  metric = MetricValues.ERRORS,
```

Comment:
updated the component to be controlled as it was not getting updated when the values change